composer.json

```diff
@@ -37,10 +37,7 @@
         "ext-ctype": "*"
     },
     "config": {
-        "vendor-dir": "includes/vendor",
-        "platform": {
-            "php": "7.4.0"
```

Comment:
Please keep the platform requirement. 
PHP must be the lowest version YOURLS and its dependencies support.

Reply:
Thanks - the lowest version YOURLS supports is automatically determined by the dependencies. In this case it would be PHP 8.x : https://github.com/auraphp/Aura.Sql/blob/6.x/composer.json#L25
I'd rather update https://github.com/YOURLS/YOURLS/blob/master/composer.json#L19 to specify that PHP 8.x is needed. Does this make sense?

Reply:
That's correct, but the goal of a platform specification is the other way around: ensure the dependencies are _not_ updated to a version that breaks minimum requirements.
That super useful for automatic dependency management.

Reply:
I see. Makes sense. I pushed an update. Note that I've not pushed an update for `includes/vendor/`